deploy/common/deploy.sh

```diff
@@ -93,7 +93,7 @@ function wait_for_central {
     start_time="$(date '+%s')"
     local deadline=$((start_time + 10*60))  # 10 minutes
     until curl_central --output /dev/null --silent --fail "https://$LOCAL_API_ENDPOINT/v1/ping"; do
-        if [[ "$(date '+%s')" > "$deadline" ]]; then
+        if [[ "$(date '+%s')" -gt "$deadline" ]]; then
```

Comment:
Agreed. I made this change because my spellcheck complained about it, but maybe it was due to a version mismatch. I'll revert this and see if CI passes